b+tree.ts

```diff
@@ -488,6 +488,213 @@ export default class BTree<K=any, V=any> implements ISortedMapF<K,V>, ISortedMap
     return {nodequeue, nodeindex, leaf:nextnode};
   }
 
+  /**
+   * Computes the delta operations (adds, deletes, and modifications) required to go from this tree to the supplied destination.
+   * For efficiency, the delta is returned via invocations of supplied handlers.
+   * The computation is optimized for the case in which the two trees have large amounts of shared data (obtained by calling the
+   * `clone` or `with` APIs) and will avoid any iteration of shared state.
+   * @param destination The tree to compute a delta to.
+   * @param processAdd Callback invoked for all adds in the delta.
+   * @param processDelete Callback invoked for all deletions in the delta.
+   * @param processModify Callback invoked for all modifications in the delta.
```

Comment:
It may be worth noting that if you desire custom equality, you can implement it in your processModify call back (and not what equality is used to decided if it will invoke processModify).